packages/grid/_modules_/grid/components/panel/GridPanelWrapper.tsx

```diff
@@ -18,5 +22,9 @@ export function GridPanelWrapper(
 ) {
   const classes = useStyles();
   const { className, ...other } = props;
-  return <div className={classnames(classes.root, className)} {...other} />;
+  return (
+    <TrapFocus open disableEnforceFocus isEnabled={() => true} getDoc={() => document}>
```

Comment:
getDoc is used to get the active element is the window the data grid is rendered. The current logic fails if rendered in a popup, iframe, else. @dtassone do you want the grid to support windows different from the main one?

Reply:
yes we should support iframe popup...
Storybook uses iframe

Reply:
> Storybook uses iframe

In storybook, the iframe is the main window. It's not the same case.

Reply:
Ok, then we can create a new issue for it. We don't support it, e.g. enable the iframe in the first demo and see how it renders:

```diff
diff --git a/docs/src/pages/components/data-grid/overview/overview.md b/docs/src/pages/components/data-grid/overview/overview.md
index 41ee8b90..3aaf44ca 100644
--- a/docs/src/pages/components/data-grid/overview/overview.md
+++ b/docs/src/pages/components/data-grid/overview/overview.md
@@ -29,7 +29,7 @@ This abstraction also set constraints that allow the component to implement new
 import { DataGrid } from '@material-ui/data-grid';
 ```

-{{"demo": "pages/components/data-grid/overview/DataGridDemo.js", "defaultCodeOpen": false, "bg": "inline"}}
+{{"demo": "pages/components/data-grid/overview/DataGridDemo.js", "defaultCodeOpen": false, "bg": "inline", "iframe": true}}

 ### Commercial version [<span class="pro"></span>](https://material-ui.com/store/items/material-ui-pro/)
```

<img width="830" alt="Screenshot 2021-02-23 at 14 58 33" src="https://user-images.githubusercontent.com/3165635/108854079-bd5fcf00-75e7-11eb-916b-e5fda9e0477d.png">

Reply:
keyboard navigation is not working, menu container is not the current window, arrow alignment is wrong. I can imagine there are more issues.

Reply:
Ok this is because of the `getDoc={() => document}` part and if we load it in an iframe right? Should we expand the use case to make it work. I didn't catch it in the docs nor in the storybook demos.

Reply:
I think that we should ignore the problem, until it's surfaced by actual users. 

I'm confident that if we need to solve the problem, we will technically be able to. Even in the core, I have recently fixed one of the component support for it. It's rare developers ask for it.

If we take the arcade game analogie, this is a level 100 boss, we are still at level 25.

Reply:
I like the analogy 😅